packages/components/src/unit-control/types.ts

```diff
@@ -51,14 +52,12 @@ export type UnitSelectControlProps = {
 	isUnitSelectTabbable?: boolean;
 	/**
 	 * A callback function invoked when the value is changed.
-	 *
-	 * @default noop
 	 */
 	onChange?: UnitControlOnChangeCallback;
 	/**
 	 * Size of the control option. Supports "default" and "small".
 	 *
-	 * @default 'default'
+	 * @default default
```

Comment:
Oh I was wondering about this "quote or no quote" problem too! It does look better this way in the props table, but I wonder if that's just a quirk in the docgen/Storybook or it's actually the correct JSDoc format. My google-fu failed me there.

Reply:
I thought about it a bit more, and I've decided to add the quotes back (it feels like the correct thing to do).

We may want to open an issue in the Storybook repo (or the ts docgen repo) to improve the way default string values are displayed?